### PR TITLE
New Parameter DeferRegistrationWhenPackagesAreInUse

### PR DESCRIPTION
**Pull request by RDMacLachlan**
Included examples and description for the new parameter "-DeferRegistrationWhenPackagesAreInUse"

### CODE REVIEW

**Comment by e0i** [comment]
@dariomws @get-itips @tseward

Peer review needed for this PR. Thanks.

**Comment by e0i** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

**Review by tseward — APPROVED** [review]

File: docset/windows/appx/Add-AppxPackage.md [hunk 1]
```diff
@@ -94,7 +95,14 @@ PS C:\> Add-AppxPackage -Path "C:\Users\user1\Desktop\MyApp.msix" -DependencyPat
 
 This command adds an app package that the package contains.
 
-### Example 2: Add a disabled app package in development mode
+### Example 2: Update an app, but defer registration until app has closed.
```
Comment (JohanFreelancer9) [inline docset/windows/appx/Add-AppxPackage.md:98]:
```suggestion
### Example 2: Update an app, but defer registration until the app has closed

```

File: docset/windows/appx/Add-AppxPackage.md [hunk 2]
```diff
@@ -498,6 +506,21 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
+### -DeferRegistrationWhenPackagesAreInUse
+Specifies that the app will not register for a user if currently in use. The app will update on next launch.
```
Comment (JohanFreelancer9) [inline docset/windows/appx/Add-AppxPackage.md:510]:
```suggestion
Specifies that the app will not register for a user if currently in use. The app will update on the next launch.

```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by get-itips — APPROVED** [review]

File: docset/windows/appx/Add-AppxPackage.md [hunk 3]
```diff
@@ -31,7 +31,8 @@ Adds a signed app package to a user account.
 Add-AppxPackage [-Path] <String> [-DependencyPath <String[]>] [-RequiredContentGroupOnly]
  [-ForceApplicationShutdown] [-ForceTargetApplicationShutdown] [-ForceUpdateFromAnyVersion]
  [-RetainFilesOnFailure] [-InstallAllResources] [-Volume <AppxVolume>] [-ExternalPackages <String[]>]
- [-OptionalPackages <String[]>] [-RelatedPackages <String[]>] [-WhatIf] [-Confirm] [<CommonParameters>]
+ [DeferRegistrationWhenPackagesAreInUse] [-OptionalPackages <String[]>] [-RelatedPackages <String[]>]
```
Comment (dariomws) [inline docset/windows/appx/Add-AppxPackage.md:34]:
```suggestion
 [-DeferRegistrationWhenPackagesAreInUse] [-OptionalPackages <String[]>] [-RelatedPackages <String[]>] 

```

**Review by dariomws — APPROVED** [review]
I'm ok after this minor detail is fixed. Thanks!

**Comment by RDMacLachlan** [comment]
All changes have been applied to the Doc. Thank you for reviewing, and providing your comments.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by RDMacLachlan** [comment]
I'm unable to press the **Merge pull request** button as I'm not authorized. Can someone please merge this request or provide me with access to. Thanks!

**Pull request merged**